routes/pizza.js

```diff
@@ -1,11 +1,35 @@
 /* set up the router */
+const express = require('express');
+const indexRoute = express.Router();
```

Comment:
Why is it called indexRoute? Maybe just name it `app` or `router` to more generically describe what it is?